kuberenetes/standalone-deployment.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: mongoose
+  namespace: mongoose
+spec:
+  selector:
+    matchLabels:
+      name: mongoose
+  template:
+    metadata:
+      labels:
+        name: mongoose
+    spec:
+      containers:
+        - image: emcmongoose/mongoose
+          name: mongoose-controller
+          command: ["/bin/sh"]
```

Comment:
probably not needed

Reply:
Yes, it is really optional. I saw such a record from the guys from the flex.